plugins/chat/app/controllers/api/chat_threads_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Chat::Api::ChatThreadsController < Chat::Api
+  def show
+    render_serialized(
```

Comment:
Obviously extremely dumb, no security checks etc, just to get the idea across, will not merge in this state.